main.rb

```diff
@@ -0,0 +1,89 @@
+require_relative 'app'
+
+# rubocop:disable Metrics/CyclomaticComplexity
+def main
+  app = App.new
+  puts 'Welcome to the Library Management App!'
+
+  loop do
+    display_menu
+    choice = gets.chomp.to_i
+
+    case choice
+    when 1 then app.list_all_books
+    when 2 then app.list_all_people
+    when 3 then create_teacher(app)
+    when 4 then create_student(app)
```

Comment:
- [ ] Well done applying the changes required by the previous CR but the app should match the original design and there are extra options that don't exist in the original design, please 🙏🏽 refactor your code to add people with the options teacher or student to match the original design.